doc/index.rst

```diff
@@ -25,20 +25,30 @@ as simple as:
 
 This will create a job called ``myrobot`` on your machine, which launches
 base.launch. It will start automatically when you next start your machine,
-or you can bring it up and down manually:
+or you can bring it up and down manually (command may differ per providers (``upstart`` by default)):
 
 .. code-block:: bash
 
+    When upstart is in use
     $ sudo service myrobot start
     $ sudo service myrobot stop
 
+    systemd
+    $ sudo systemd start myrobot
+    $ sudo systemd stop myrobot
```

Comment:
Should be `sudo systemctl` right?

Reply:
@mikepurvis Common mistake/typo. You're right. Fixed.